dialogs/simplelink.js

```diff
@@ -1,7 +1,13 @@
 ﻿(function (CKEDITOR) {
     'use strict';
 
-    var containsScheme = /:\/\//;
+    function containsScheme (href) {
+        return /^[a-zA-Z]+:/.test(href);
+    }
+
+    function isEmail (href) {
+        return href.indexOf('@') !== -1;
```

Comment:
Fixed. There's obviously tons of email regexes out there, but the one provided by Regular-Expressions.info seems fairly trustworthy with good reasoning for it's compromises without being extremely complex.